Utils/github_workflow_scripts/handle_external_pr.py

```diff
@@ -64,7 +67,26 @@ def determine_reviewer(potential_reviewers: List[str], repo: Repository) -> str:
     return selected_reviewer
 
 
-def get_packs_support_level_label(file_paths: List[str]) -> str:
+def get_packs_support_levels(pack_dirs: Set[str]) -> Set[str]:
```

Comment:
`Set`s don't retain order.
Wouldn't you rather use `list`s, or return a `dict`, so you could match `pack_dir` to `pack_support_level`?

Reply:
i don't really care about order here, its enough to get the support levels as a set and print out packs that we couldn't get their support labels.